docs/deployment/README.md

```diff
@@ -18,58 +18,62 @@ The deployment documentation is current for the following component versions:
+* Before deploying the Cumulus software, a CloudFormation stack is deployed that creates necessary [IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) via the `iams` stack.
+* The Cumulus software is configured and deployed via the `app` stack.
 
 --------------
+
 ## Requirements
 
 ### Linux/MacOS software requirements
 
-- git
-- [node 8.10](https://nodejs.org/en/) (use [nvm](https://github.com/creationix/nvm) to upgrade/downgrade)
-- [npm](https://www.npmjs.com/get-npm)
-- sha1sum or md5sha1sum
-- zip
-
-- AWS CLI - [AWS command line interface](https://aws.amazon.com/cli/)
-- python
+* git
+* [node 8.10](https://nodejs.org/en/) (use [nvm](https://github.com/creationix/nvm) to upgrade/downgrade)
+* [npm](https://www.npmjs.com/get-npm)
+* sha1sum or md5sha1sum
+* zip
+* AWS CLI - [AWS command line interface](https://aws.amazon.com/cli/)
+* python
 
 ### Credentials
 
 * [CMR](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/common-metadata-repository) username and password.  Can be excluded if you are not exporting metadata to CMR.
-
 * [EarthData Client login](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/earthdata-login) username and password. User must have the ability to administer and/or create applications in URS.  It's recommended to obtain an account in the test environment (UAT).
 
-
 ### Needed Git Repositories
 
-- [Cumulus](https://github.com/nasa/cumulus) (optional)
-- [Cumulus Dashboard](https://github.com/nasa/cumulus-dashboard)
-- [Deployment Template](https://github.com/nasa/template-deploy)
-
+* [Cumulus](https://github.com/nasa/cumulus) (optional)
+* [Cumulus Dashboard](https://github.com/nasa/cumulus-dashboard)
+* [Deployment Template](https://github.com/nasa/cumulus-template-deploy)
 
 ## Installation
 
 ### Prepare DAAC deployment repository
 
-_If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration. ](deployment-readme#prepare-aws-configuration)_
+_If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration.](deployment-readme#prepare-aws-configuration)_
 
 Clone template-deploy repo and name appropriately for your DAAC or organization
 
-    $ git clone https://github.com/nasa/template-deploy <daac>-deploy
+```bash
+  $ git clone https://github.com/nasa/template-deploy <daac>-deploy
+```
 
 Enter repository root directory
 
-    $ cd <daac>-deploy
+```bash
+  $ cd <daac>-deploy
+```
 
 Then run:
 
-    $ npm install
+```bash
+  $ nvm use
```

Comment:
In nasa/cumulus-template-deploy#13, I added an `.nvmrc` to the template-deploy repo, so adding a step to `nvm use` here

Reply:
👍